cli/specifier_handler.rs

```diff
@@ -24,13 +24,8 @@ use std::pin::Pin;
 use std::sync::Arc;
 
 pub type DependencyMap = HashMap<String, Dependency>;
-pub type FetchFuture = Pin<
-  Box<
-    (dyn Future<Output = Result<CachedModule, (ModuleSpecifier, AnyError)>>
-       + 'static
-       + Send),
-  >,
->;
+type FetchFutureOutput = Result<CachedModule, (ModuleSpecifier, AnyError)>;
+pub type FetchFuture = Pin<Box<dyn Future<Output = FetchFutureOutput> + Send>>;
```

Comment:
This `Send` seems fishy as well - we're using a single threaded runtime after all

Reply:
It's not trivially removable ...